src/main/java/com/khutircraftubackend/exception/GlobalExceptionHandler.java

```diff
@@ -222,4 +224,10 @@ public ResponseEntity<String> handleGenericException(Exception ex) {
                 .body("Глобальне повідомлення: Внутрішня помилка сервера. Будь ласка, спробуйте пізніше.");
     }
 
+    @ExceptionHandler(IOException.class)
+    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
+    public String handleIOException(IOException e) {
+        return "Failed to upload file: " + e.getMessage();
```

Comment:
first look at this class suggests we have all response messages in Ukrainian, 
but that suddenly seems is not the case, is it?

Reply:
user should probably get response with some generic message but server might need to log it out on console and maybe into a file as well.

Reply:
Corrected.